scrapper.py

```diff
@@ -150,9 +161,15 @@ def manage(self, slaves):
             #task: (client_addr, url)
             with availableSlaves.get_lock():
                 if availableSlaves.value > 0:
-                    task = socketPull.recv_json()
-                    log.debug(f"Pulled {task[1]} in worker:{self.uuid}")
-                    taskQueue.put(task)
+                    addr, url = socketPull.recv_json()
```

Comment:
he should receive here the `uuid` of client, to find his address in `self.clients`